plugins/hydra_ray_launcher/hydra_plugins/hydra_ray_launcher/conf/hydra/launcher/ray_aws.yaml

```diff
@@ -134,15 +135,11 @@ hydra:
         setup_commands:
           # TODO clean up set up commands.
           # Ray is only officially supported up to python 3.7
-          - conda create -n hydra37 python=3.7 -y
-          - echo 'export PATH="$HOME/anaconda3/envs/hydra37/bin:$PATH"' >> ~/.bashrc
-          - pip install -U ray
-          - pip install -U https://hydra-test-us-west-2.s3-us-west-2.amazonaws.com/hydra_core-1.0.0rc1-py3-none-any.whl
-          - pip install -U https://hydra-test-us-west-2.s3-us-west-2.amazonaws.com/hydra_ray_launcher-0.1.0-py3-none-any.whl
-          # Consider uncommenting these if you also want to run apt-get commands during setup
-          # - sudo pkill -9 apt-get || true
-          # - sudo pkill -9 dpkg || true
-          # - sudo dpkg --configure -a
+          - conda create -n hydra38 python=3.8 -y
```

Comment:
1. I suggest calling the conda env hydra regardless of the python version.
2. are we expecting the users to reuse the same environment more than once? are we expecting them to change the python version they use mid-stream?